ui/src/app/applications/components/application-sync-options.tsx

```diff
@@ -33,10 +33,46 @@ function booleanOption(name: string, label: string, defaultVal: boolean, props:
 
 const optionStyle: React.CSSProperties = {marginTop: '0.5em'};
 
+interface SyncOption {
+    name?: string;
+    key: SyncOptionKey;
+    value?: boolean;
+    default?: boolean;
+}
+
+const enum SyncOptions {
+    Validate = 'Validate',
```

Comment:
We have only following sync options: `PruneLast=true`, `Validate=false`and `CreateNamespace=true`. 

* DryRun can be specified in UI/API during manual sync only
* ApplyOnly  is UI only checkbox that allows to specify. SyncStrategy can be specified in UI/API during manual sync only
* Force is specified as a boolean field in of sync strategy. Probably is should be replaced with a sync option but this can should be done in a separate PR.